src/main/interface/msp.c

```diff
@@ -1141,6 +1141,8 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, flight3DConfig()->deadband3d_low);
         sbufWriteU16(dst, flight3DConfig()->deadband3d_high);
         sbufWriteU16(dst, flight3DConfig()->neutral3d);
+        sbufWriteU16(dst, flight3DConfig()->limit3d_low);
```

Comment:
We'll leave the configurator settings out for now, until it has been confirmed that this is the way that this should be done.

Reply:
Ok, I just followed the code for `deadband3d_low` . If you are suggesting changes, I do not know what to change.

Reply:
You should revert the changes to `src/main/interface/msp.c`. The way that new features are added to Betaflight is that they are made available in CLI only in a first stage, so that users can test them, and propose fixes / improvements. Once a final form has been established, and it is not likely for the parameters to change any more, they are added to MSP and the configurator.